server/src/test/java/org/eclipse/openvsx/adapter/VSCodeAdapterTest.java

```diff
@@ -246,6 +247,15 @@ public void testAssetNotFound() throws Exception {
                 .andExpect(status().isNotFound());
     }
 
+    @Test
+    public void testAssetVsixPackage() throws Exception {
+        mockExtension();
+        mockMvc.perform(get("/vscode/asset/{namespace}/{extensionName}/{version}/{assetType}",
+                "redhat", "vscode-yaml", "0.5.2", "Microsoft.VisualStudio.Services.VSIXPackage"))
+                .andExpect(status().isOk())
+                .andExpect(header().string(HttpHeaders.CONTENT_TYPE, "application/zip"));
```

Comment:
Shouldn't it be `"application/vsix"`?

Reply:
Just checked it in MS marketplace, in this case is `application/zip` :thinking:

Reply:
Ok, let's use `application/zip`